pages/about.html

```diff
@@ -0,0 +1,140 @@
+    <p>We take the operation of our various <a href="/services/">services</a> very seriously, and require all participants to adhere to our <a href="https://github.com/privacytoolsIO/.github/blob/master/CODE_OF_CONDUCT.md">Code of Conduct</a>. For any questions or to report abuse, please see our CoC’s <a href="https://github.com/privacytoolsIO/.github/blob/master/CODE_OF_CONDUCT.md#enforcement">Enforcement section</a>.</p>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col">
+    <h2>Core Contributors</h2>
+  </div>
+</div>
+
+<div class="row">
+
+{% include team.html
+avatar="burung.jpg"
+name="Burung Hantu"
+role="Founder"
+bio="I founded PrivacyTools in 2015 to share resources and tools, allowing users to avoid international mass-surveillance programs."
+email="dude@privacytools.io"
+mastodon="https://social.privacytools.io/@BurungHantu"
+twitter="privacytoolsIO"
+%}
+
+{% include team.html
+avatar="jonah.png"
+name="Jonah Aragon"
+nick="Jonah"
+role="Administrator"
+bio="I run the website and services for PrivacyTools. My goal is to spread the word about data privacy as widely as possible."
+email="jonah@privacytools.io"
+website="https://www.jonaharagon.com"
+mastodon="https://social.privacytools.io/@jonah"
+twitter="JonahAragon"
+blog="jonah"
+%}
+
+{% include team.html
+avatar="blacklight447.png"
+name="blacklight447"
+bio="I research new privacy recommendations and moderate our communities. My expertise is endpoint security and networking."
+role="Community Manager"
+email="blacklight447@privacytools.io"
+mastodon="https://social.privacytools.io/@blacklight447"
+%}
+
+{% include team.html
+avatar="danarel.jpg"
+name="Dan Arel"
+role="ThinkPrivacy Editor"
+bio="I'm an author, columnist, and privacy advocate whose work has appeared in Time, Huff Post, OpenSource, and more."
+website="https://www.danarel.com"
+twitter="danarel"
+mastodon="https://mastodon.technology/@danarel"
+blog="danarel"
+%}
+
+{% include team.html
+avatar="dawid.png"
+name="Dawid Potocki"
+role="Developer"
+bio="I'm a student interested in software development. I help improve PrivacyTools and promote using free libre software."
+website="https://dawidpotocki.com"
+email="dawidpotocki@privacytools.io"
+mastodon="https://social.privacytools.io/@dawidpotocki"
+%}
+
+{% include team.html
+avatar="dngray.png"
+name="dngray"
+nick="Daniel Gray"
+bio="I research and communicate with privacy-focused service providers to refine our recommendations, like email and VPN providers."
+email="dngray@privacytools.io"
+%}
+
+{% include team.html
+avatar="mikaela.png"
+name="Mikaela Suomalainen"
+nick="Mikaela"
+bio="I am interested in instant messengers. I review privacy tools, and participate in our forum, matrix room, and issue tracker."
```

Comment:
```suggestion
bio="I am interested in instant messengers. I review privacy tools, and participate in our forum, Matrix room, and issue tracker."
```

I am not sure how much this reflects me, but I guess I can PR-change it anytime, but I can surely say that names are written with a capital letter.